### PR TITLE
Load picture in ViewMoodFragment

### LINKED ISSUE #95: Fix: Load picture in ViewMoodFragment, Char restriction on reason
**Description:** 
An issue just to keep track of things so we don't forget.
-Currently a picture attached to a MoodEvent doesn't load when viewing the event
-No character restriction on the reason

### PR DESCRIPTION
**Pull request by jameshahaha**
Closes #95

**Summary**
- View loads images correctly now
- Changed MoodServices to return the Url of the images instead of the filepath
- Added char + word limit on Reason
- Minor comments/cleaning

### CODE REVIEW

File: Moodlet/app/src/main/res/layout/fragment_add_mood.xml [hunk 1]
```diff
@@ -184,7 +184,8 @@
                 android:layout_marginStart="16dp"
                 android:layout_marginLeft="16dp"
                 android:layout_marginTop="8dp"
-                android:text="Somebody touch spaghet"
+                android:maxLength="20"
+                android:hint="Reason"
```
Comment (WJXHenry) [inline Moodlet/app/src/main/res/layout/fragment_add_mood.xml]: Please change this to "Somebody touch spaghet" and make it a string resource

**Review by WJXHenry — CHANGES_REQUESTED** [review]

File: Moodlet/app/src/main/res/layout/fragment_add_mood.xml (hunk 1 above)
Comment (jameshahaha) [reply Moodlet/app/src/main/res/layout/fragment_add_mood.xml]: ok dad

**Review by WJXHenry — APPROVED** [review]

**Comment by WJXHenry** [comment]
LGTM

**Comment by jameshahaha** [comment]
LGTM

image

**Comment by jameshahaha** [comment]
LGTM
image

**Pull request merged**